scripts/core/exp-command-buffer.yml

```diff
@@ -84,6 +84,10 @@ etors:
             [uint32_t] Reference count of the command-buffer object.
             The reference count returned should be considered immediately stale.
             It is unsuitable for general use in applications. This feature is provided for identifying memory leaks.
+    - name: IS_UPDATABLE
+      desc: |
+            [$x_bool_t] Returns the value of $x_exp_command_buffer_desc_t::isUpdatable
+            that the command-buffer was created with.
```

Comment:
Preference for this query returning all of the `$x_exp_command_buffer_desc_t` struct rather than just the individual member, if possible, as it is a more extensible design if we want to query other parts of it in future. If it's a pain to reconstruct this in the adapters code though it's not a big deal though.

Reply:
I've gone with the latter approach for now (returning false if ignored), but can easily change to storing the unsused properties if that's your preference.